goal_src/jak2/engine/load/loader.gc

```diff
@@ -25,5 +93,2377 @@
+                             (22 . wait)
+                             (23 . wait)
+                             (24 . wait)
+                             (25 . wait)
+                             (27 . wait)
+                             (26 . wait)
+                             (28 . wait)
+                             (31 . wait)
+                             (35 . wait)
+                             )
+          )
+    (set! (-> gp-0 cmd 27) '((64 . wait)
+                             (21 . wait)
+                             (22 . wait)
+                             (23 . wait)
+                             (24 . wait)
+                             (25 . wait)
+                             (27 . wait)
+                             (26 . wait)
+                             (28 . wait)
+                             (31 . wait)
+                             (35 . wait)
+                             )
+          )
+    (set! (-> gp-0 cmd 26) '((64 . wait)
+                             (21 . wait)
+                             (22 . wait)
+                             (23 . wait)
+                             (24 . wait)
+                             (25 . wait)
+                             (27 . wait)
+                             (26 . wait)
+                             (28 . wait)
+                             (31 . wait)
+                             (35 . wait)
+                             )
+          )
+    (set! (-> gp-0 cmd 28) '((64 . wait)
+                             (21 . wait)
+                             (22 . wait)
+                             (23 . wait)
+                             (24 . wait)
+                             (25 . wait)
+                             (27 . wait)
+                             (26 . wait)
+                             (28 . wait)
+                             (31 . wait)
+                             (35 . wait)
+                             )
+          )
+    (set! (-> gp-0 cmd 31) '((64 . wait)
+                             (21 . wait)
+                             (22 . wait)
+                             (23 . wait)
+                             (24 . wait)
+                             (25 . wait)
+                             (27 . wait)
+                             (26 . wait)
+                             (28 . wait)
+                             (31 . wait)
+                             (35 . wait)
+                             )
+          )
+    (set! (-> gp-0 cmd 35) '((64 . wait)
+                             (21 . wait)
+                             (22 . wait)
+                             (23 . wait)
+                             (24 . wait)
+                             (25 . wait)
+                             (27 . wait)
+                             (26 . wait)
+                             (28 . wait)
+                             (31 . wait)
+                             (35 . wait)
+                             )
+          )
+    gp-0
+    )
+  )
```

Comment:
what the heck is this

Reply:
The numbers are actually `gui-channel`s. I alMost want to write a hack for thos function so that it prints those instead.